tools/btfgen.sh

```diff
@@ -21,24 +21,26 @@ fi
 # We do not need to add network eBPF programs like dns or snisnoop here because
 # they do not rely on BTF (i.e., they do not include vmlinux.h).
 ${BTFHUB}/tools/btfgen.sh -a ${ARCH}                                                                \
-    -o ${INSPEKTOR_GADGET}/pkg/gadgets/audit-seccomp/tracer/auditseccomp_bpfel.o                    \
-    -o ${INSPEKTOR_GADGET}/pkg/gadgets/audit-seccomp/tracer/auditseccompwithfilters_bpfel.o         \
-    -o ${INSPEKTOR_GADGET}/pkg/gadgets/bindsnoop/tracer/core/bindsnoop_bpfel.o                      \
-    -o ${INSPEKTOR_GADGET}/pkg/gadgets/biotop/tracer/biotop_bpfel.o                                 \
-    -o ${INSPEKTOR_GADGET}/pkg/gadgets/capabilities/tracer/core/capabilities_bpfel.o                \
+    -o ${INSPEKTOR_GADGET}/pkg/gadgets/audit-seccomp/tracer/auditseccomp_bpfel_x86.o                \
+    -o ${INSPEKTOR_GADGET}/pkg/gadgets/audit-seccomp/tracer/auditseccompwithfilters_bpfel_x86.o     \
+    -o ${INSPEKTOR_GADGET}/pkg/gadgets/bindsnoop/tracer/core/bindsnoop_bpfel_x86.o                  \
+    -o ${INSPEKTOR_GADGET}/pkg/gadgets/biotop/tracer/biotop_bpfel_x86.o                             \
+    -o ${INSPEKTOR_GADGET}/pkg/gadgets/capabilities/tracer/core/capabilities_bpfel_x86.o            \
     -o ${INSPEKTOR_GADGET}/pkg/gadgets/execsnoop/tracer/core/execsnoop_bpfel.o                      \
-    -o ${INSPEKTOR_GADGET}/pkg/gadgets/filetop/tracer/filetop_bpfel.o                               \
-    -o ${INSPEKTOR_GADGET}/pkg/gadgets/fsslower/tracer/core/fsslower_bpfel.o                        \
+    -o ${INSPEKTOR_GADGET}/pkg/gadgets/filetop/tracer/filetop_bpfel_x86.o                           \
```

Comment:
> Could we include bpf_tracing.h in all bpf sources so that it is consistent?

Gadgets using [bpf-tracing.h](`https://github.com/libbpf/libbpf/blob/29869d6ef0893cc1c907705f919fdfd6feb7cc13/src/bpf_tracing.h`) need to be compiled for each different architecture. This PR proposes a solution where gadgets that don't use this file are compiled only two times: big and little endian machines. Finally, btfgen.sh is removed in https://github.com/kinvolk/inspektor-gadget/pull/710 to avoid having to add each single gadget there.